### PR TITLE
ProofPoint Threat Response

### PR DESCRIPTION
**Pull request by bakatzir**
fixes: https://github.com/demisto/etc/issues/13355

new integration, developed on costumer env, was not tested.

DO NOT MERGE

### CODE REVIEW

File: Integrations-NotTested/integrations-proofpoint_threat_response.yml [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+description: 'Proofpoint builds lists for enforcement actions '
+detaileddescription: |-
+  In order to create API Key
+  Settings -> API Key -> Add api key
+configuration:
+- display: Server URL (e.g https://uxnysecthreat01.homebox.com)
```
Comment (NeelRust) [inline Integrations-NotTested/integrations-proofpoint_threat_response.yml]: @bakatzir need to have a better default domain name,

File: Integrations-NotTested/integrations-proofpoint_threat_response.yml [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+
+        message = ''
+        for member in members:
+            add_to_list(list_id, member, None, expiration)
+            message += '{} added successfully to block_hash list\n'.format(member)
+
+        demisto.results(message)
+
+
+    def search_members(list_id, member_filter):
```
Comment (NeelRust) [inline Integrations-NotTested/integrations-proofpoint_threat_response.yml]: What entity do we refer to as member and lists here?

Comment (bakatzir) [reply Integrations-NotTested/integrations-proofpoint_threat_response.yml]: list_id refers to a specific blacklist in the product, each list contains different values(a list for urls, another for hashes, etc)
member - a member of the list, can be anything, depends on the list.

Comment (anara123) [reply Integrations-NotTested/integrations-proofpoint_threat_response.yml]: @bakatzir change member terminology to indicator

Comment (bakatzir) [reply Integrations-NotTested/integrations-proofpoint_threat_response.yml]: Done

File: Integrations-NotTested/integrations-proofpoint_threat_response.yml (hunk 1 above)
Comment (bakatzir) [reply Integrations-NotTested/integrations-proofpoint_threat_response.yml]: True, did it.

**Pull request merged**